testing/trino-testing-containers/src/main/java/io/trino/testing/containers/OpenTracingCollector.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.testing.containers;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import static com.google.common.net.UrlEscapers.urlFragmentEscaper;
+
+public class OpenTracingCollector
+        extends BaseTestContainer
+{
+    private static final Path storageDirectory;
+
+    static {
+        try {
+            storageDirectory = Files.createTempDirectory("tracing-collector");
+        }
+        catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    private static final int COLLECTOR_PORT = 4317;
+    private static final int HTTP_PORT = 16686;
+
+    public OpenTracingCollector()
+    {
+        super(
+                "jaegertracing/all-in-one:latest",
+                "opentracing-collector",
+                Set.of(COLLECTOR_PORT, HTTP_PORT),
+                Map.of(),
+                Map.of(
+                    "COLLECTOR_OTLP_ENABLED", "true",
+                    "SPAN_STORAGE_TYPE", "badger", // KV that stores spans to the disk
+                    "GOMAXPROCS", "2"), // limit number of threads used for goroutines
+                Optional.empty(),
+                1);
+
+        withRunCommand(List.of(
+                "--badger.ephemeral=false",
+                "--badger.span-store-ttl=15m",
+                "--badger.directory-key=/badger/data",
+                "--badger.directory-value=/badger/data",
+                "--badger.maintenance-interval=30s"));
+
+        withCreateContainerModifier(command -> command.getHostConfig()
+                .withMemory(2 * 1024 * 1024 * 1024L)); // 1 GB limit
```

Comment:
should the comment say "2 GB" or should the number be 1 GB?

Reply:
https://github.com/trinodb/trino/pull/18307